Dashboard dates

jmuharsky opened the pull request:
Add created_date and modified_date to dashboards.

klausw commented:
How does this handle pre-existing dashboards which don't have associated dates? Do they initially start out blank, and then pick up a "last modified" date when changed while leaving creation date blank? Would be nice if there were a way to backfill values from datastore metadata.

Otherwise LGTM.

jmuharsky commented:
Unfortunately, there's no metadata to go off of.  The behavior you're describing is accurate -- they'll get the created and modified dates set on the first put(), and modified will be updated for every successive put().
